### ISSUE TITLE
Showing suggestions once the text field is focuse

### ISSUE THREAD

**Issue by sAlfaifi** [dab246/super_tag_editor#47]
Hello,
Thank you for working on this package.
Is it possible to show all suggestions when I focus on the text field?

**Comment by dab246** [comment]
> Hello, Thank you for working on this package. Is it possible to show all suggestions when I focus on the text field?

Of course, if the list of suggestions you pass in is not empty. If you have any other ideas, I would welcome your contributions to this project. Thanks.

**Comment by balasubramanian1612s** [comment]
Great work man!

I have a query,

The suggestions gets triggered when there is a change in text. But I want the suggestions to be shown immediately when the field is focused. Is there anyway possible?
